templates/themes/app/pagelayout.html.twig

```diff
@@ -1,24 +1,32 @@
-{% extends '@ibexadesign/pagelayout_variables.html.twig' %}
+{% extends '@ibexadesign/pagelayout/variables.html.twig' %}
 
 {% block pagelayout_content %}
 <!DOCTYPE html>
 <html lang="{{ app.request.locale|replace({'_': '-'}) }}">
 <head>
     <meta charset="utf-8">
 
-    {% block page_head %}
-        {% include '@ibexadesign/page_head.html.twig' %}
+    {% block page_head_title %}
+        {% include '@ibexadesign/pagelayout/head/title.html.twig' %}
+    {% endblock %}
+
+    {% block page_head_meta %}
+        {% include '@ibexadesign/pagelayout/head/meta.html.twig' %}
+    {% endblock %}
+
+    {% block page_head_link %}
+        {% include '@ibexadesign/pagelayout/head/link.html.twig' %}
     {% endblock %}
 
     {% block page_head_style %}
-        {% include '@ibexadesign/page_head_style.html.twig' %}
+        {% include '@ibexadesign/pagelayout/head/style.html.twig' %}
     {% endblock %}
 
     {% block page_head_script %}
-        {% include '@ibexadesign/page_head_script.html.twig' %}
+        {% include '@ibexadesign/pagelayout/head/script.html.twig' %}
     {% endblock %}
 
-    {% block open_graph %}
+    {% block page_head_opengraph %}
```

Comment:
I would not rename Twig blocks.

This will make upgrades from older media site versions unnecessarily harder as these block names are used in Netgen Layouts in Twig blocks, and other projects might use more of them.